core/unit_test/CMakeLists.txt

```diff
@@ -124,6 +124,7 @@ foreach(Tag Threads;Serial;OpenMP;Cuda;HPX;OpenMPTarget;HIP)
     foreach(Name
       TeamBasic
       TeamReductionScan
+      TeamScan
```

Comment:
Just remove this test for `SYCL`. There is a list further down for that.

Reply:
Ah, I had a lot of trouble finding it but just needed to rebase